source/testing/testing-controllers.md

```diff
@@ -101,55 +106,43 @@ export default Controller.extend({
 });
 ```
 
-This time when we setup our `moduleFor` we need to pass an options object as
-our third argument that has the controller's `needs`.
-
-```tests/unit/controllers/comments-test.js
-import { moduleFor, test } from 'ember-qunit';
-import EmberObject from '@ember/object';
-import { run } from '@ember/runloop';
-
-moduleFor('controller:comments', 'Comments Controller', {
-  needs: ['controller:post']
-});
-```
-
 Now let's write a test that sets a property on our `post` model in the
 `PostController` that would be available on the `CommentsController`.
 
 ```tests/unit/controllers/comments-test.js
-import { moduleFor, test } from 'ember-qunit';
-import EmberObject from '@ember/object';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import EmberObject from "@ember/object";
 import { run } from '@ember/runloop';
 
-moduleFor('controller:comments', 'Comments Controller', {
-  needs: ['controller:post']
-});
+module('Unit | Controller | comments', function(hooks) {
+  setupTest(hooks);
 
-test('should modify the post model', function(assert) {
-  assert.expect(2);
+  test('should modify the post model', function(assert) {
+    assert.expect(2);
 
-  // grab an instance of `CommentsController` and `PostController`
-  const ctrl = this.subject();
-  const postCtrl = ctrl.get('post');
+    // grab an instance of `CommentsController` and `PostController`
+    let controller = this.owner.lookup('controller:comments');
+    let postCtrl = controller.get('post');
 
-  // wrap the test in the run loop because we are dealing with async functions
-  run(function() {
+    // wrap the test in the run loop because we are dealing with async functions
```

Comment:
this comment is very misleading since there isn't a single async/await in this test 🤔